admin-ui/app/directives/ups-client-snippets.html

```diff
@@ -3,21 +3,21 @@
       ng-init="state.activeSnippet = typeEnum[variant.type].snippets[0]"
       ng-class="{ active: snippet == state.activeSnippet }">
     <a href ng-click="state.activeSnippet = snippet">
-      <span ng-if="snippet == 'android'">Android</span>
-      <span ng-if="snippet == 'simplePush'">JavaScript</span>
-      <span ng-if="snippet == 'ios_objc'">ObjectiveC</span>
-      <span ng-if="snippet == 'ios_swift'">Swift</span>
-      <span ng-if="snippet == 'dotnet'">C#</span>
-      <span ng-if="snippet == 'cordova'">Cordova</span>
-      <span ng-if="snippet == 'titanium'">Titanium</span>
-      <span ng-if="snippet == 'adm'">ADM</span>
-      <span ng-if="snippet == 'push_config'">push-config.json</span>
+      <span ng-if="snippet == 'android'" translate>Android</span>
```

Comment:
requiring i18n support in ups-admin-ui for FeedHenry. they use it to manage the push notification.